README.md

```diff
@@ -106,7 +106,7 @@ jwt.sign({
 
 ### jwt.verify(token, secretOrPublicKey, [options, callback])
 
-(Asynchronous) If a callback is supplied, function acts asynchronously. Callback is passed the decoded payload if the signature and optional expiration, audience, or issuer are valid. If not, it will be passed the error.
+(Asynchronous) If a callback is supplied, function acts asynchronously. The callback is passed the decoded payload if the signature is valid and optional expiration, audience, or issuer are valid. If not, it will be passed the error.
 
```

Comment:
Good change, however, this sentence always feels weird to me, don't you think it should use the same "pattern" as we use for `sign`? 
**The callback is called with the** decoded payload... . If not, it will be **called with** the error.

Reply:
Updated. I also made the wording between the Synchronous and Asynchronous section the same.